src/SourceBuild/content/README.md

```diff
@@ -71,59 +71,83 @@ For the latest information about Source-Build support, please watch for announce
 
 ### Prerequisites
 
-The dependencies for building .NET from source can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md).
+The dependencies for building can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/).
```

Comment:
should we have this here? I remember you filed an issue about needing node/npm for aspnetcore which the runtime doc doesn't capture

Reply:
The VMR needs to list its requirements in a better way, yes. But this is a separate issue. I'm just changing the link to the OS invariant docs landing page in runtime.

Reply:
Yep. I just found your node issue and it already mentions we should update the readme: https://github.com/dotnet/source-build/issues/3924